manifests/plugin/openscap/params.pp

```diff
@@ -6,4 +6,7 @@
   $listen_on               = 'https'
   $openscap_send_log_file  = '/var/log/foreman-proxy/openscap-send.log'
   $spooldir                = '/var/spool/foreman-proxy/openscap'
+  $contentdir              = '/usr/share/foreman-proxy/openscap/content'
```

Comment:
this is embarrassing. It was `/var/lib/` all along. Updating
ref [#1](https://github.com/theforeman/foreman-packaging/blob/rpm/1.11/rubygem-smart_proxy_openscap/rubygem-smart_proxy_openscap.spec#L6), [#2](https://github.com/theforeman/foreman-packaging/blob/rpm/1.11/rubygem-smart_proxy_openscap/rubygem-smart_proxy_openscap.spec#L88-L91)